### PR TITLE
Fixes #80

### LINKED ISSUE #80: Global InlineSVGModule.forRoot configuration does not work in lazy loaded modules
Reproduction:
- https://stackblitz.com/edit/angular-r9cqys
- https://angular-r9cqys.stackblitz.io

When calling `InlineSVGModule.forRoot({ baseUrl: 'https://dev.w3.org/SVG/tools/svgweb/samples/svg-files' })` inside `app.module.ts`, the `baseUrl` is not carried into lazy modules when there is no `inlineSVG` directive used inside non-lazy loaded components.

In the reproduction, navigate to the lazy loaded route and check then network panel. You will see that requests for SVGs are made against `/` instead of the base URL that I provided `https://dev.w3.org/SVG/tools/svgweb/samples/svg-files`.

You can work around the issue by using an `inlineSVG` directive or injecting the `SVGCacheService` somewhere in a non-lazy loaded component. Then you will see that the SVG in the lazy loaded component is actually made against the correct base URL.

Note that the SVG requests in the repro are not working due to CORS issues, but focus on the request URL rather than the actual response.

### PR DESCRIPTION
**Pull request by nathanosdev**
Make all services singletons by providing them only through the forRoot method.

### CODE REVIEW

File: demo/src/bootstrap.ts [hunk 1]
```diff
@@ -12,9 +12,9 @@ enableProdMode();
 
 @NgModule({
   declarations: [DemoComponent],
-  imports: [BrowserModule, HttpClientModule, InlineSVGModule],
+  imports: [BrowserModule, HttpClientModule, InlineSVGModule.forRoot()],
   bootstrap: [DemoComponent]
 })
-class DemoAppModule {}
+class DemoAppModule { }
```
Comment (dudyn5ky1) [inline demo/src/bootstrap.ts:18]: I think empty `{}` should be without space. This is not related to the change in this PR, but is good to consider and/or update `tslint` config with this

**Review by dudyn5ky1 — APPROVED** [review]

File: demo/src/bootstrap.ts (hunk 1 above)
Comment (JovicaConkic) [reply demo/src/bootstrap.ts:18]: Don't worry. Microsoft allowed it.

Comment (arkon) [reply demo/src/bootstrap.ts:18]: Not sure what Microsoft has to do with it, but I'm fine with this.

File: src/svg-cache.service.ts [hunk 2]
```diff
@@ -1,81 +1,80 @@
-import { Injectable, Optional, Renderer2, RendererFactory2 } from '@angular/core';
+import { Injectable, Optional, Renderer2, RendererFactory2, Inject } from '@angular/core';
```
Comment (arkon) [inline src/svg-cache.service.ts]: Minor nitpick: can you keep these sorted alphabetically?

**Review by arkon — CHANGES_REQUESTED** [review]

File: demo/src/bootstrap.ts (hunk 1 above)
Comment (nathanosdev) [reply demo/src/bootstrap.ts:18]: He's making a reference to Microsoft's purchase of Github. Best to ignore this anyway 😄

File: src/svg-cache.service.ts [hunk 3]
```diff
@@ -1,81 +1,80 @@
-import { Injectable, Optional, Renderer2, RendererFactory2 } from '@angular/core';
+import { Inject, Injectable, Optional, Renderer2, RendererFactory2 } from '@angular/core';
```
Comment (nathanosdev) [inline src/svg-cache.service.ts:1]: Sorted alphabetically now

File: demo/src/bootstrap.ts (hunk 1 above)
Comment (dudyn5ky1) [reply demo/src/bootstrap.ts:18]: @arkon in some parallel universe Github's acquisition is related to code style in `ng-*` directives, so that's fine.

**Review by arkon — APPROVED** [review]

**Comment by arkon** [comment]
Thanks!

**Comment by arkon** [comment]
Published as v8.0.0.

**Comment by nathanosdev** [comment]
Awesome, thanks for being so responsive!

**Pull request merged**